build.gradle

```diff
@@ -43,6 +43,9 @@ dependencies {
     compile "io.grpc:grpc-netty:1.+"
     compile "io.grpc:grpc-testing:1.+"
     compile "com.xenomachina:kotlin-argparser:2.0.4"
+    compile 'com.google.cloud:google-cloud-storage:1.14.0'

```

Comment:
replace the last numbers with `+` ? Things change so frequently these days...

Reply:
done